packages/devextreme/js/__internal/ui/chat/chat_header.ts

```diff
@@ -0,0 +1,65 @@
+import type { Properties } from '@js/core/dom_component';
+import DOMComponent from '@js/core/dom_component';
+import type { dxElementWrapper } from '@js/core/renderer';
+import $ from '@js/core/renderer';
+
+const CHAT_HEADER_CLASS = 'dx-chat-header';
+const CHAT_HEADER_TEXT_CLASS = 'dx-chat-header-text';
+
+export interface ChatHeaderProperties extends Properties {
+  title?: string;
+}
+
+class ChatHeader extends DOMComponent<ChatHeaderProperties> {
+  private _$text?: dxElementWrapper;
+
+  _getDefaultOptions(): ChatHeaderProperties {
+    return {
+      // @ts-expect-error
+      ...super._getDefaultOptions(),
+      ...{
+        title: '',
+      },
+    } as ChatHeaderProperties;
+  }
+
+  _init(): void {
+    // @ts-expect-error
+    super._init();
+
+    $(this.element()).addClass(CHAT_HEADER_CLASS);
+  }
+
+  _initMarkup(): void {
+    // @ts-expect-error
+    super._initMarkup();
+
+    this._renderText();
+  }
+
+  _renderText(): void {
+    const { title } = this.option();
+
+    this._$text = $('<p>')
```

Comment:
Why should we use `<p>` element?

Reply:
[90dd3acd3c6dd8b06c21608a237d774ee8b6aa43]